src/app/CompilerPane.py

```diff
@@ -696,35 +701,49 @@ async def make_comp_widgets(frame: ttk.Frame) -> None:
     )
     vrad_frame.grid(row=1, column=0, sticky='ew')
 
+    UI['light_none'] = ttk.Radiobutton(
+        vrad_frame,
+        text=gettext('None'),
+        value='NONE',
+        variable=vrad_compile_type,
+    )
+    UI['light_none'].grid(row=0, column=0)
     UI['light_fast'] = ttk.Radiobutton(
         vrad_frame,
         text=gettext('Fast'),
-        value=0,
-        variable=vrad_light_type,
+        value='FAST',
+        variable=vrad_compile_type,
     )
-    UI['light_fast'].grid(row=0, column=0)
+    UI['light_fast'].grid(row=0, column=1)
     UI['light_full'] = ttk.Radiobutton(
         vrad_frame,
         text=gettext('Full'),
-        value=1,
-        variable=vrad_light_type,
+        value='FULL',
+        variable=vrad_compile_type,
     )
-    UI['light_full'].grid(row=0, column=1)
+    UI['light_full'].grid(row=0, column=2)
+
+    def light_conf_swap(switch_to: str): 
```

Comment:
The `gettext()` function parameter needs to be a literal, because they're read out of the source code for the translation files. It'd be better if this was just two variables.